pkgs/applications/networking/kubo/default.nix

```diff
@@ -2,15 +2,15 @@
 
 buildGoModule rec {
   pname = "kubo";
-  version = "0.22.0"; # When updating, also check if the repo version changed and adjust repoVersion below
+  version = "0.23.0"; # When updating, also check if the repo version changed and adjust repoVersion below
   rev = "v${version}";
 
-  passthru.repoVersion = "14"; # Also update kubo-migrator when changing the repo version
```

Comment:
Migrator was updated at commit https://github.com/NixOS/nixpkgs/commit/888e3d7d5a0556ff0929802214a5d1f5e3b85839

Merged to the main branch at commit https://giAthub.com/NixOS/nixpkgs/commit/7c190346f14a0dd6bf8c00a32f681de53a598ea7

Reply:
The URL has a typo but thanks for linking it!